pkg/cedar/intake/schema_versions.go

```diff
@@ -0,0 +1,21 @@
+package intake
+
+// SchemaVersion is a human-readable version for the schemas EASi sends to the CEDAR Intake API
```

Comment:
Can you add a comment in here noting that when we update the structs in `/intake/models` that we should update the appropriate schema version?

Reply:
I thought about this a bit; really, what needs to happen is that if a struct gets updated in `intake/models`, the developer also needs to add a new version in `schema_versions.go`, as well as updating `cmd/gen_intake_schema/main.go` to use the new version. I can't figure a way of programmatically enforcing this, though. I'm worried that putting info just in the intake README and/or `schema_versions.go` will be overlooked, so maybe the best solution is a comment in each model file under `intake/models`? What do you think?

Reply:
In https://github.com/CMSgov/easi-app/pull/1450/commits/a754e47a18d249b65be9b31425d25ada75cff642, I added notes in each model file to update the schema version when changes are made to the types.